avocado/core/result.py

```diff
@@ -125,7 +125,7 @@ def start_tests(self):
         """
         Called once before any tests are executed.
         """
-        self.tests_run += 1
+        pass
```

Comment:
I always disliked this, but had no time to investigate it thoroughly. Thanks :smile: